plutus-playground-client/test/MainFrameTests.purs

```diff
@@ -4,6 +4,7 @@ module MainFrameTests
 
 import Prelude
 
+import AjaxUtils (ourDecodeJson)
```

Comment:
can't this just be called `decodeJson`? it's namespaced to `AjaxUtils`

Reply:
Yeah, good point. 👍